apcd-cms/src/apps/registrations/templates/submission_form/registration_form_scripts.html

```diff
@@ -1,4 +1,12 @@
-<aside>
+<aside>    
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```

Comment:
This jQuery load seems like it doesn't need to be copied over. This page does not have the jQuery that https://github.com/TACC/Core-CMS-Custom/pull/195/files#diff-8555cd54ebd055a9434e6c60c435fe58b59ecc6facdad1fde497a0b7b97d7c72R152-R153 does.

<sup>P.S. [jQuery (at an older version) is already loaded by Core-CMS](https://github.com/TACC/Core-CMS/blob/v3.11.6/taccsite_cms/templates/assets_core_delayed.html#L6).</sup>
<sup>P.P.S. [The jQuery code](https://github.com/TACC/Core-CMS/blob/v3.11.6/taccsite_cms/templates/assets_core_delayed.html#L6) could be written without jQuery with minimal changes. Out of scope, I know.</sup>

Reply:
Updated and removed!